src/main/groovy/com/ullink/gradle/nunit/NUnitPlugin.groovy

```diff
@@ -9,6 +9,10 @@ class NUnitPlugin implements Plugin<Project> {
         project.apply plugin: 'nunit-base'
 
         Task defaultNUnitTask = project.task('nunit', type: NUnit)
+
+        Task reportingTask = project.task('nunitReport', type: ReportGenerator)
+        defaultNUnitTask.finalizedBy(reportingTask)
```

Comment:
it might not be working for every project, as if opencover is enabled, nunit is not executed but opencover will invoke nunit instead

Reply:
As discussed, subscribing to both OpenCover and Nunit tasks. Thank you for the really good remark!